pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -261,7 +265,7 @@ func NewApiServerDeployment(namespace string, repository string, imagePrefix str
 	container := &deployment.Spec.Template.Spec.Containers[0]
 	container.Command = []string{
 		VirtAPIName,
-		"--port",
+		PortCmdLineOption,
```

Comment:
I am not sure if this makes things more readable. I would consider leaving the string in this case.